README.md

```diff
@@ -187,6 +190,64 @@ The SDK also provides a deterministic sampler and more span processing options.
 
 [Set up the Honeycomb OpenTelemetry SDK for Java](https://docs.honeycomb.io/getting-data-in/java/opentelemetry-distro/#using-the-honeycomb-sdk-builder).
 
+### Multi-span Attributes with the Baggage Span Processor
```

Comment:
I've created #169 to always add the BaggageProcessor when configuring the SDK 👍🏻